<launch>
    <arg name="model"/>
    <arg name="gui" default="false"/>
    <param name="robot_description" textfile="$(find vdm)/urdf/line.urdf" />
    <!--<node
        name="joint_state_publisher"
        pkg="joint_state_publisher"
        type="joint_state_publisher"
        required="true"
    />-->
    <node
        name="robot_state_publisher"
        pkg="robot_state_publisher"
        type="robot_state_publisher"
        required="true"
    />
    <node
        name="line_joint_state_publisher"
        pkg="vdm"
        type="line_joint_state_publisher.py"
        required="true"
    />
    <node
        name="line_marker_publisher"
        pkg="vdm"
        type="line_marker_publisher.py"
        required="true"
    />

    <node 
        name="rviz" 
        pkg="rviz" 
        type="rviz"
        required="true"   
    />
</launch>
